tests/virt/node/migration_and_maintenance/test_workload_disruption_migration.py

```
@@ -0,0 +1,255 @@
+
+
+@pytest.fixture()
+def migrated_vm_with_hotplug_support(vm_with_hotplug_support):
+    migrate_vm_and_verify(
+        vm=vm_with_hotplug_support,
+        timeout=TIMEOUT_30MIN if "windows" in vm_with_hotplug_support.name else TIMEOUT_15MIN,
+        check_ssh_connectivity=True,
+    )
+
+
+@pytest.fixture()
```

Ruff (0.15.7)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)